packages/core/src/sessionManagement.ts

```diff
@@ -1,22 +1,24 @@
 import { cacheCookieAccess, COOKIE_ACCESS_DELAY, CookieCache } from './cookie'
+import { monitor } from './internalMonitoring'
 import { Observable } from './observable'
 import { tryOldCookiesMigration } from './oldCookiesMigration'
 import * as utils from './utils'
 
 export const SESSION_COOKIE_NAME = '_dd_s'
-export const EXPIRATION_DELAY = 15 * utils.ONE_MINUTE
+export const SESSION_EXPIRATION_DELAY = 15 * utils.ONE_MINUTE
```

Comment:
Wouldn't it make sense to move those time related constants to a module called `time` instead of `utils`?

Reply:
IMO, as long as we don't duplicate these values everywhere it is fine.
utils is not that big for now, if we feel the need to split this file or to extract some of the functions at some point, we could do it.